external/model-preparation-algorithm/mpa_tasks/apis/config.py

```diff
@@ -77,6 +77,44 @@ class BaseLearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING
         )
 
+        use_adaptive_interval = configurable_boolean(
+            default_value=False,
+            header="Use adaptive validation interval",
+            description="Depending on the size of iteration per epoch, adaptively update the validation interval and related values.",
+            warning="This will automatically control the patience and interval when early stopping is enabled.",
+            affects_outcome_of=ModelLifecycle.TRAINING
+        )
+
+        enable_early_stopping = configurable_boolean(
+            default_value=True,
+            header="Enable early stopping of the training",
+            description="Early exit from training when validation accuracy isn't \
+                         changed or decreased for several epochs.",
+            affects_outcome_of=ModelLifecycle.TRAINING
+        )
+        
+        patience = configurable_integer(
```

Comment:
I think this is too general a variable name. This is the patience of early stopping, but it seems to be used together elsewhere.

Reply:
Do you suggest to change the variable name? Like early_stop_patience/early_stop_iteration_patience?

Reply:
Yes, I think that is appropriate.